bond/bond.go

```diff
@@ -42,6 +42,7 @@ type bondingConfig struct {
 	FailOverMac int                      `json:"failOverMac"`
 	Miimon      string                   `json:"miimon"`
 	Links       []map[string]interface{} `json:"links"`
+	MTU         string                   `json:"mtu"`
```

Comment:
should this be int? in the doc you mark it as int

Reply:
done